gsa/src/web/components/dashboard/dashboard.js

```diff
@@ -255,12 +255,13 @@ export class Dashboard extends React.Component {
       maxRows = DEFAULT_MAX_ROWS,
       ...props
     } = this.props;
+
     const rows = this.getRows();
 
     if (isDefined(error) && !isLoading) {
       return (
         <RowPlaceHolder>
-          {_('Could not load dashboard settings. Reason: {{error}}', {error})}
+          {_(`Could not load dashboard settings. Reason: ${error.message}`)}
```

Comment:
I don't think your change it translatable. Therefore it should be
```suggestion
          {_(`Could not load dashboard settings. Reason: {{error}}`, {error: error.message})}
```